doc/ref_arch/kubernetes/chapters/chapter02.md

```diff
@@ -20,20 +20,155 @@
+| `req.inf.ntw.15` | Network | The Architecture **should** support Service Mesh Interface (SMI). |
+| `req.inf.ntw.16` | Network | The Architecture **should**  support co-exisence of multiple Conatiner Network Interface (CNI). |
+| `req.inf.acc.01` | Acceleration | The Architecture **should** support Application Specific Acceleration. |
+| `req.inf.acc.02` | Acceleration | The Architecture **should** support NFVI Acceleration (such as SmartNICs). |
+| `req.inf.vir.01`   | Virtualisation |   The Architecture **must** support the capability for Containers to consume virtualised compute, storage and network resources.|
+| `req.inf.phy.01`  |  Physical Infrastructure |   The Architecture **must** support the capability for Containers to consume physical (aka bare metal) compute, storage and network resources. |
+
+<p align="center"><b>Table 2-2:</b> Kubernetes Architecture: Infrastructure Requirements</p>
+Please note that "shared" is a reference to multi-tenant support and pooled storage resources.
+
+<a name="2.3.3"></a>
+### 2.3.3 Kubernetes Cluster Requirements
+
+| Ref # | sub-category | Description |
+|----|------------------------------------------------|---------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
+| `req.kcm.01` | General | The Architecture **must** allow infrastructure resource sharing within a Kubernetes cluster. |
+| `req.kcm.02` | General | The Architecture **must** support discoverability of nodes and their features. |
+| `req.kcm.03` | General | The Architecture **must** support scheduling of workloads based on Enhanced Platform Awareness (EPA) features such as CPU Pinning, huge-pages and SR-IOV. |
+| `req.kcm.04` | General | The Architecture **must** include kubernetes artefacts (e.g., images, Helm charts, etc.) repository capabilities. |
+| `req.kcm.05` | General | The Architecture **should** support multi-tenancy in Kubernetes cluster. |
+| `req.kcm.06` | General | The Architecture **must** support resource tagging. |
+| `req.kcm.07` | General | The Architecture **must** support policy driven horizontal auto-scaling of Kubernetes cluster. |
+| `req.kcm.08` | General | The Architecture **must** support workload resiliency. |
+
+<p align="center"><b>Table 2-3:</b> Kubernetes Architecture: Kubernetes Cluster Requirements</p>
+
+<a name="2.3.4"></a>
+### 2.3.4 Interfaces & APIs
+
+| Ref # | sub-category | Description |
+|----|------------------------------------------------|---------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
+| `req.int.api.01` | API | The Architecture **must** provide Control API endpoints to cloud platform core services. |
+| `req.int.api.02` | API | The Architecture **must** provide APIs needed to discover and manage NFVI resources. |
```

Comment:
What is meant with NFVI resources here? Is there the assumption that the underlying infrastructure is RA1?

Reply:
@pgoyal01 could you comment this out please, I've added to the backlog (#615) as there is an open conversation.